demos/continuous_batching/README.md

```diff
@@ -45,26 +67,28 @@ node: {
   }
 }
 
-cp graph.pbtxt meta-llama/Llama-2-7b-chat-hf/
-
-tree meta-llama/
-meta-llama/
-└── Llama-2-7b-chat-hf
-    ├── config.json
-    ├── generation_config.json
-    ├── graph.pbtxt
-    ├── openvino_detokenizer.bin
-    ├── openvino_detokenizer.xml
-    ├── openvino_model.bin
-    ├── openvino_model.xml
-    ├── openvino_tokenizer.bin
-    ├── openvino_tokenizer.xml
-    ├── special_tokens_map.json
-    ├── tokenizer_config.json
-    ├── tokenizer.json
-    └── tokenizer.model
+cp graph.pbtxt Llama-2-7b-chat-hf/
```

Comment:
is this dir from optimum-cli command? if iso it will be under model_server/Llama-2-7b-chat-hf
and we are checking model_server/demos/continuous_batching/Llama-2-7b-chat-hf

move "cd demos/continuous_batching" before optimum-cli command?